tools/cli/src/commands/operator.rs

```diff
@@ -0,0 +1,45 @@
+use crate::context::AppContext;
+use anyhow::Result;
+use lavs_mock_operators::msg::{AllVotersResponse, QueryMsg};
+use layer_climb::prelude::*;
+
+pub struct OperatorQuerier {
+    pub ctx: AppContext,
+    pub contract_addr: Address,
+    pub querier: QueryClient,
+}
+
+impl OperatorQuerier {
+    pub async fn new(ctx: AppContext, contract_addr: Address) -> Result<Self> {
+        let querier = QueryClient::new(ctx.chain_config.as_ref().clone()).await?;
+        Ok(Self {
+            ctx,
+            contract_addr,
+            querier,
+        })
+    }
+
+    pub async fn all_operators(&self) -> Result<Vec<Operator>> {
+        let all_voters: AllVotersResponse = self
+            .querier
+            .contract_smart(&self.contract_addr, &QueryMsg::AllVoters {})
+            .await?;
+
+        all_voters
+            .voters
+            .into_iter()
+            .map(|v| {
+                let address = self.ctx.chain_config.parse_address(&v.address)?;
```

Comment:
Huh, why do we need to parse the address rather than leave it as a string?

Oh, for sending directly to it, not for displaying. Right?

Reply:
hmm... good catch... it's more a subjective preference than a hard requirement here. I could make it just pass `VoterInfo` through (which is `Uint128` for `power` and `String` for `address`)

This was a small bit of an unnecessary transformation, but it does feel like the right thing to do to me since I feel like we'll eventually want to use the `Operator` for other things like querying balances. Can't really justify it on a pure requirements level though, I can get rid of it and just return `VoterInfo` if you want.

Reply:
I see it's approved so merging for now, but please feel free to request a change here if you want, no worries